server/db/utils/hash.js

```diff
@@ -1,10 +1,11 @@
 const crypto = require('crypto');
 
 const secret = process.env.HASH_SECRET || 'aBadCryptoSecret';
+const saltySecret = secret + process.env.SALT;

```

Comment:
we would need to salt the user's password; not our HASH_SECRET: then run it through our hash function